static/docs/commands-reference/destroy.md

```diff
@@ -43,11 +43,58 @@ $ ls -a
 
 $ dvc destroy
 
-This will destroy all information about your pipelines as well as cache in .dvc/cache.
+This will destroy all information about your pipelines, all data files, as well as cache in .dvc/cache.
 Are you sure you want to continue?
 yes
 
 $ ls -a
 
 .git code.py foo
 ```
+
+#### Cache directory is changed from default
+
+By default, the cache location is `.dvc/cache`. Let's change the cache location
```

Comment:
check the https://dvc.org/doc/commands-reference/remote-add to see how to split example properly into blocks. You should use multiple "```dvc" sections with some text in between.